lib/parser/prescan.cc

```diff
@@ -391,11 +391,24 @@ bool Prescanner::NextToken(TokenSequence &tokens) {
     }
     if (*at_ == ' ' || *at_ == '\t') {
       // Compress free-form white space into a single space character.
-      // Discard white space at the end of a line.
       const auto theSpace{at_};
+      char previous{at_ <= start_ ? ' ' : at_[-1]};
       NextChar();
       SkipSpaces();
-      if (*at_ != '\n') {
+      if (*at_ == '\n') {
+        // Discard white space at the end of a line.
+      } else if (!inPreprocessorDirective_ &&
+          (previous == '(' || *at_ == '(' || *at_ == ')')) {
```

Comment:
What about when `previous` is ')'? As in: `use m, only: operator(+) , operator(-)`

We can skip the space after the ')' either here or when parsing. But skipping it here matches the description of "Discard whitespace before/after parentheses".

Reply:
Any space after `)` is skipped by the parser and doesn't need to be squashed by the prescanner.  I improved the comment.